packages/rehype/package.json

```diff
@@ -21,6 +21,9 @@
     "rehype-stringify": "^7.0.0",
     "unified": "^9.0.0"
   },
+  "devDependencies": {
+    "hast-util-to-html": "^7.1.1"
+  },
```

Comment:
This shouldn’t be here, should it? 🤔

Reply:
If this is omitted, `lerna` won’t install `hast-util-to-html`, and TypeScript can’t allow its types.

It’s used [here](https://github.com/rehypejs/rehype/pull/35/files#diff-18f766a2f6dfc9f6661bdfd3e2d0e65dR5)

Alternatively, the following could be defined in `remark-stringify`.

```ts
type RemarkStringifyOptions = HastUtilToHtmlOptions
```

Then that aliased type could be used by remark and this dev dependency could be removed.

Reply:
Ahh. I think I like `RehypeStringifyOptions`, for it similarity to `RehypeParserOptions`. to abstract away what’s underneath?